README.md

```diff
@@ -71,12 +71,19 @@ with:
   caCertificate: ${{ secrets.VAULTCA }}
 ```
 - **github**: you must provide the github token as `githubToken`
+
+**Notice: [Vault GitHub authentication](https://www.vaultproject.io/docs/auth/github)
+requires `read:org` permissions for authentication. The auto-generated `GITHUB_TOKEN`
+created for projects does not have these permissions and GitHub does not allow this
+token's permissions to be modified. A new GitHub Token secret must be created with
```

Comment:
> A new GitHub Token secret must be created...

I think such token can only be created as personal access tokens so wondering if we should call that out explicitly. FWIW there's a similar notice on the GitHub auth docs page, with the potential security downside stated as well: https://www.vaultproject.io/docs/auth/github.